src/lib/layout/network-menu/NetworkAccordion.tsx

```diff
@@ -19,6 +19,7 @@ interface NetworkAccordionProps {
   setCursor: (index: Option<number>) => void;
   startIndex: number;
   onClose: () => void;
+  isEditable?: boolean;
 }
 
 export const NetworkAccordion = ({
```

Comment:
I am thinking about creating a new component `NetworkAccordionLocal.tsx` which doesn't have the subsection components (I believe the design team says that there is no subsection for local)